internal/sign/ocpbuild/maker.go

```diff
@@ -0,0 +1,255 @@
+package ocpbuild
+
+import (
+	"bytes"
+	"context"
+	"embed"
+	"fmt"
+	"text/template"
+
+	"github.com/mitchellh/hashstructure/v2"
+	buildv1 "github.com/openshift/api/build/v1"
+	"github.com/rh-ecosystem-edge/kernel-module-management/internal/api"
+	"github.com/rh-ecosystem-edge/kernel-module-management/internal/constants"
+	buildutils "github.com/rh-ecosystem-edge/kernel-module-management/internal/utils/build"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/utils/pointer"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+type TemplateData struct {
```

Comment:
I don't see anyone outside this package using `TempalteData`. Any reason it is exported?